### DISCUSSION TITLE
2024-06-14 - Effects of Anchoring - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#8732] [Chewing the Fat]
https://www.ssw.com.au/rules/recognize-anchoring-effects/

How do you feel about anchoring inside where you work?
Does it happen much?
Any good ideas to avoid it?

**Comment by Hona** [comment]
Its important to be aware of biases and try to work with them

**Comment by miichaelsmedley** [comment]
I think anchoring is one of the most important skills a developer needs to learn when talking to stakeholders, especially about the two most important things: time and money. These two factors are intrinsically related. Understanding and managing these anchors can significantly impact the success of any project.

**Comment by ricksu978** [comment]
stay open minded and flexible for changes always bring more opportunities

**Reply by MarcusIrmscherSSW** [reply]
It's always good to remain open to alternative ideas. If you're anchored to one thing, who knows what you might miss out on.💡

**Comment by calumjs** [comment]
I watch a lot of air crash investigation videos, and often the accident is caused by pilots getting anchored on what they think the problem is and interpreting all new evidence in that context rather than stepping back and re-analysing the situation.

**Comment by matt-goldman-ssw** [comment]
The anchoring effect is one of many cognitive biases covered in [You Are Not So Smart](https://www.amazon.com.au/You-Are-Not-So-Smart/dp/1592407366), one of may favourite books of all time. It's like the missing manual for your brain, and I highly recommend everyone reads it.

Awesome rule. As for the video...I think I prefer to get my info from David McRaney (author of the above-mentioned book) than from Lex Luthor. Oh, I'm sorry,
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/169313794/d45b9403-6a7a-4d41-82c5-d785838806cf)

**Reply by jeoffreyfischer** [reply]
Roasting Jeff Bezos and providing an Amazon URL in the same message..

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/106663901/ff1c6756-ae88-4682-98bc-5c0b886fc854)

**Reply by matt-goldman-ssw** [reply]
@jeoffreyfischer lexcorpbooks.com was down for maintenance

**Reply by GordonBeeming** [reply]
I've added it to my library on audible... I guess also amazon 😅
https://www.audible.com.au/pd/You-Are-Not-So-Smart-Audiobook/B00FGA9MOO

**Reply by GordonBeeming** [reply]
Another book which is good for understanding the phycology around how these sub-conscious triggers can benefit conversations/sales/anchoring to your advantage is [Influence, New and Expanded -
The Psychology of Persuasion](https://www.audible.com.au/pd/Influence-New-and-Expanded-Audiobook/0063097796?action_code=ASSGB149080119000H&share_location=pdp)

Without going through You Are Not So Smart yet, I'm assuming part of the benefit is the same... that you are more aware of these different sub-conscious behaviors

**Reply by Calinator444** [reply]
Ironically hearing Jeff Bezos explain the impact of tribalism in our decision-making process probably would have made me less receptive to the idea if I wasn't already. I just haven't been a fan of him or Amazon as a business in general after hearing about the conditions their employees work in.

https://www.forbes.com/sites/katherinehamilton/2023/05/24/delivery-drivers-sue-amazon-for-being-forced-to-pee-in-bottles/

**Comment by camillars** [comment]
Being unbiased in a conversation requires a lot of effort to stay open to new ideas and suggestions. It's definitely an area we can all improve 😊

**Comment by Calinator444** [comment]
I think SSW generally encourages diversity of thought. I can't speak to the experience of others but I've never felt uncomfortable sharing my opinions at SSW, even as an intern. My ideas aren't always correct and occasionally they get shot down, but I expressing your, ideas having them challenged, and correcting your assumptions is an important part of learning.

On another note one thing this video fails to address is the Hawthorne affect. If a Junior is sitting in a meeting with Adam when they come up with an idea for instance they might be less inclined to share an idea out of fear of being wrong and looking unprofessional.

**Reply by matt-goldman-ssw** [reply]
This is a really interesting perspective. It's an fun social experiment looking at how different people have interpreted the core message here. Anyway in regard to your last comment (in your edit), you should _definitely_ read You Are Not So Smart. There's an interesting commentary on this (you may have heard me talking about it on the podcast).

**Reply by Calinator444** [reply]
@matt-goldman-ssw  you're gonna get me to read that book eventually aren't you? and yes, I heard you talking about it in your podcast 😆

**Comment by TobyChurches** [comment]
Its definitely not easy to remain unbiased during a discussion, but it can also be intimidating at times to voice your opinion, especially when you have a new/opposing idea to that of the senior developers in the discussion. I really like the idea of making the juniors speak first. It makes people actively think about the idea without just blindly agreeing with the more senior devs ideas.

**Comment by leahy268** [comment]
SSW is pretty good at allowing everyone to speak.  Everyone is usually encouraged to speak their mind which is refreshing.

Did anyone else notice that video didn't mention Anchoring once?

**Reply by matt-goldman-ssw** [reply]
No, because we were anchored to thinking it was about anchoring

**Reply by GordonBeeming** [reply]
It covered it from a perspective that if senior people talk first, they'd anchor others which can lead to less diverse input into conversations as most people would talk only if they think similar to what's been said by seniors in the team

**Comment by BenDunkerleySSW** [comment]
Anchoring is usually an inevitable stage of decision making. We naturally want to prioritise or consider the biggest, "most valuable" piece of information, and have to force ourselves to look at other details too - taking a step back to observe the whole field.

**Comment by alexleessw** [comment]
I think anchoring happens a lot, consciously or subconsciously. It's about being self-aware and open minded 

Bezos has some guns 💪

**Comment by drwharris** [comment]
There are 2 sides to anchoring. What we are talking about, avoiding creating biases. But someone has to go first and that creates some bias. The other part not talked about is people should think critically in these situations and be prepared to (respectfully) question and challenge what I put forward. On the flip side, people should be prepared to be questioned and challenged, and the hardest part, say, no actually, I was wrong.

**Comment by GabbyWard** [comment]
I think this is an excellent topic to bring to everyone's attention (including myself). While I have experienced anchoring in many of my conversations, both work and non-work related, I never really identified it. This will help me going forward think of it when going into these conversations in the future. 👍

Now, the topic is good, but I am unsure why this was the video to exemplify anchoring? I found it too long to get to the point and that it didn't really address the topic of anchoring that much, with only one example that kinda covered it coming to mind. Also, I like the key takeaways in the rule, but I would like them fleshed out more in the rule itself, potentially even with more examples (whether written or maybe more media material).

Regardless of the above, I actually found a lot of value in this CTF and I'll be looking into cognitive bias a bit more going forward. 🧠

**Comment by AlexRothwell** [comment]
I've found that where you're able, getting everyone's opinions captured without the bias of others and then later discussing works quite well. A great example of this I've used in the past is planning poker: https://www.mountaingoatsoftware.com/agile/planning-poker

**Comment by awaltos** [comment]
Being aware of our biases is very important. They could be existing ones as well as anchoring to what was just said by someone older/smarter/more confident/etc. This is why planning poker (although sometimes long and tedious) is a good way to avoid this problem - at least in the first round of estimating a task; anchoring can come right back in when re-estimating.
The suggestion of letting juniors go first is good as long as there is an environment that supports people speaking up and not feeling pressured or anxious to say the 'right thing'. This is a difficult thing to achieve especially in new teams but can improve over time.

**Comment by ncn-ssw** [comment]
Asking good questions is an art, and I recommend the book ["Find Out Anything From Anyone, Anytime: Secrets of Calculated Questioning from a Veteran Interrogator" by James Pyle and Maryann Karinch](https://www.amazon.com.au/Find-Out-Anything-Anyone-Anytime-ebook/dp/B0C94WP9NT).

**Comment by jimmy1992abc** [comment]
Anchoring can be much stronger if the tone is set by a senior. It is also widely used by journalists to guide the interviewee to express what the host wants to show the audience. It can be very hard to challenge or break free from this influence, making it likely that you will end up feeling been strung alone.

I think the best way to overcome anchoring is to say, "No, I don't agree." Doing so may require bravery and a sound argument.

When you want to lead the conversation, such as when talking to clients, be confident and "feel credible enough" to give yourself a higher chance to set the tone.

**Comment by jeoffreyfischer** [comment]
This is the first time I heard about anchoring - it's a powerful concept.
It's all about asking relevant questions before providing an answer. 
Coming from a Researcher background, I've always naturally processed this way.
I've learned that it applies to not just research, but to most life and work situations.

**Comment by bettybondoc** [comment]
I think anchoring is something we all do, often without realising it. We all have opinions (informed or otherwise) and tend to express them to our colleagues—sometimes assertively if it's a strong opinion (and it becomes an anchor), or more casually if it's a minor 2c.

More than that though, a more important consideration for me is the proper definition of hierarchy (who speaks first). The most "senior" person in the room shouldn't be determined merely by age, total years of experience, or company tenure. Instead, seniority should be defined by expertise in the specific field of interest. For example, in discussions about server virtualisation techniques, I wouldn't dare presume to be more senior than a Junior SysAdmin or even a SysAdmin intern. Similarly, I feel pretty confident challenging design decisions, even if the anchor has been set by the most senior Solution Architect in the room (even Adam 😉).

As for avoiding being anchored, it's about finding the right balance between active listening and maintaining a healthy skepticism.

**Comment by cvionnet** [comment]
The example in the rule is excellent as it tackles what is probably the trickiest question to answer: "How long will it take? And how much will it cost?"

Anchoring is present in our lives in many ways, and it's good to be able to recognize it.

**Comment by awaltos** [comment]
📖 Loving all the book suggestions 😄

**Comment by JackDevAU** [comment]
Good work on the rule! One of the better ones!  💪 

Everyone has biases, and as you get to know people better, you can determine their specific biases before engaging in a conversation. 
From my experience at SSW, every individual has an opportunity to be heard, regardless of their level of experience. I think we house a pretty good environment! 

But its good to be aware of it and a good time to reflect for people if it has happened recently

**Comment by Tanya-Leahy** [comment]
Hearing everyone's opinion is important to avoid creating bias. I've found this concept a refreshing change at SSW. 
In my previous experience, anchoring in meetings often led to poor decisions because those most affected by the decision, and had the data to back it, couldn't share their opinions.

I agree with Betty, it's definitely a balance between active listening and maintaining a healthy skepticism. Being aware of anchoring allows us to recognize its influence and ensure that all perspectives are considered, leading to more balanced and informed decision-making.

**Comment by Tanya-Leahy** [comment]
[https://outlook-1.cdn.office.net/assets/reaction/like.png]     Tanya Leahy [SSW] reacted to your message:




[cid:ssw-circle(1)_4c7259a6-23f6-49e5-b933-230ca304562e.png]

Tanya Leahy
SSW Operations Coordinator
Stalk: SSW People<https://www.ssw.com.au/people/Tanya-Leahy?utm_source=emailsignature>   |   Walk: SSW Brisbane<https://www.ssw.com.au/ssw/Company/Offices/?utm_source=emailsignature>   |   Talk: +61 423 441 680<https://www.ssw.com.au/ssw/Company/ContactUs.aspx>
[cid:youtube-icon_b8b5cda9-a973-4973-8763-cf3d4922e060.png] <https://www.youtube.com/c/SSWTV>    [cid:linkedin-icon_b153ac9a-eb66-47d9-9c51-ad20b42b8a0a.png]  <https://www.linkedin.com/company/ssw/>    [cid:x-logo_3eb6d0d3-24fb-487d-a4a7-2e30cddc1ebf.png]  <https://x.com/SSW_TV>    [cid:teams-icon_552803c2-9c44-4267-b563-7fe1a7a0e589.png] ***@***.***>


        TinaCMS, the #1 CMS on GitHub, is now part of the SSW family<https://adamcogan.com/2024/05/15/ssw-purchases-tinacms?utm_source=emailsignature>

________________________________

**Comment by uly1** [comment]
Anchoring is also a useful negotiation technique... so sometimes you do want to speak 1st :)

**Reply by bahjatalaadelssw** [reply]
Yes if you're the demon or the flower girl 😂

**Comment by tiagov8** [comment]
I personally feel pretty open to discuss different ideas at SSW
... but I see that often design tasks are requested with design requirements. This can definitely influence HOW the work is going to be done, especially with junior designers. 

**Stakeholders** and **Devs** should aim to ask for UI/UX goals, without giving design specifics.
**Designers** must remember that even if a design task is given with specifics, they should focus on the goal and try providing the best approach.

**Comment by ozair-ashfaque** [comment]
I would love to learn more about anchoring

**Comment by bahjatalaadelssw** [comment]
We need some work on the "hard truths" discussions, as for anchoring

It's an exercise in training one's ego. My mentor taught me early in my career not to influence people because it robbed them of their opinion and insight. He said, "Even if you hated the project the least you can say is 'We worked hard on this and we hope you enjoy it'"

**Comment by amankumarrr** [comment]
I believe that the way you address someone and how you phrase your question can significantly influence their opinion, creating an anchoring effect. To minimize this bias, you can compare data from anonymous responses with data from non-anonymous responses.

**Comment by tomek-i** [comment]
the laws of human nature by Robert Greene
https://www.youtube.com/watch?v=uhWzVdGmX2w

Speaking about biases, watch this:
https://www.youtube.com/watch?v=vKA4w2O61Xo

**Comment by arig2024** [comment]
Anchoring is a powerful tool that presents both positive and negative effects or outcomes. The more you are conscious of the effects of the anchor - the more succesful you will likely be. Some philosophies talk of 'Mindfulness'. If anyone is interested in the effects this can have on life in general (& has time) have a listen to this: https://www.youtube.com/watch?v=BOJ9AY-41oE&ab_channel=FreakonomicsRadioNetwork

**Comment by Ben0189** [comment]
Anchoring isn't always a bad thing, sometimes it improve the efficiency in Decision-Making. 
more self-confidence less anchoring 😊

**Reply by Calinator444** [reply]
I'm not sure I necessarily agree. Yes if you accept an initial assumption as true it will make the decision making process quicker but what if you initial assumption was wrong?

**Comment by chrisschultzssw** [comment]
As others have said - it's important to be aware of anchoring/biases we have when making decisions.
I found the rule to be very focused on sales/estimates, but the topic is much bigger than that.

**Comment by YalalCakirSSW** [comment]
Ironically, I started this CTF with checking the github discussion first 😅 I'm not sure if I understand the point of this CTF correctly. Anchoring is a new concept to me. I read the rule and watched the video couple of times.   

Is it about...
- Importance of the first idea and the direction of the conversation?
- Juniors being able to express their thoughts freely?
- Getting influenced by others' decisions?
- Telling/finding the truth being hard?
- Sales people's approach on clients?
- Should we avoid anchor our decisions or should we use it strategically?
- Would I be aware of the anchoring effect if I get influenced from a decision? 

Personally, I care about and learn from ideas of people I respect.

**Reply by tomek-i** [reply]
I think anchoring is the australian way of calling it bias ?

I had the same issue because the video talks about bias  but SSW talks about anchoring so its either it refers to the same thing **bias** or I don't know either :-D  

 Maybe it is the australian way of calling bias,  anchoring or SSW way of trying to be hip and calling it ⚓ :-D

**Reply by YalalCakirSSW** [reply]
Still a bit unclear to me 😄

**Reply by stephanfakossw** [reply]
It's more like entrenched bias from my understanding - where a person can't get pass their initial bias

**Comment by stephanfakossw** [comment]
Being unbiased in professional interactions is key to managing expectations and relations. The only think you can really do is being persistent in trying to avoid personal bias and being patient with other people's biases.

**Comment by AntPolkanov** [comment]
I think anchoring happens always when there is money / reputation involved. All people want easy answers to their questions - "it will cost X and will be done in Y".

**Comment by Dhruv-0987** [comment]
I think not speaking first or speaking at all depends entirely on the senior members, in most cases its not a question of if you can speak or will you be wrong or not, its about the response you are expecting even when you are faaaar from being right.

For me, getting to know the right answer is more important then how I feel about it and would not say something or make a decision if its not backed up by data or proven to be right (For now).

**Comment by christoment** [comment]
> How do you feel about anchoring inside where you work?

Yes I think anchoring is very important especially when talking to a decision maker

> Does it happen much?

It depends!

> Any good ideas to avoid it?

Instead of avoiding, I think we should leverage anchoring - e.g. customer service usually sets the time to service higher than the expected outcome so customer feels happy about it! 😄

**Comment by will-truscott-ssw** [comment]
Anchoring is an inevitable and constant part of human cognition, I find the best strategies are to not avoid anchoring as a whole, but to instead figure out ways to identify and mitigate it during times where it is negatively effecting you or your team. 

As there's been a lot of anti-anchoring in this thread, I'll say something on the potential positives! 
It can sometimes be healthy for leadership to embrace this bias. Strategies like not giving information until you're certain on something or allowing more junior team members to speak first can be great in many circumstances, but can also create unclear expectations/direction while also potentially increasing anxiety around saying something that another more senior team member disagrees with you on. I personally find it's useful to phrase information as free discussion and to develop a culture where everyone feels as though they're able to speak their mind (and prompted to if you think they'd have something to add) to catch these differing opinions early and often.

**Reply by duanxinhuan** [reply]
Yes, it also helps you all the time!!!!
A prior is forever bad :)
Taking an a prior opinion seriously is stupid, lol

**Comment by duanxinhuan** [comment]
I don't like the term cognitive bias, it's BS.
The so called cognitive bias is also your personality, they help you in many cases, you just need to figure out if they can help you or trap you in different situations.

We are forever emotional animals, trying to be rational will inevitably put us into self-contradiction, double standard and eventually opinions on innocent people.

Just simply admits we hate a lot of things in our life out of "rationality", and allow the others to simply hate something is important, use rationality to deny other people's feeling is bad.

Back to the topic of client work:
The client likes you for 100 reasons just like the client hate you for 100 reasons.
Just do enough background research on them, and make unique plans based on our process.

Fun scientific fact: Rationality may not even exist, discovering cognitive bias is also a feeling.

**Reply by duanxinhuan** [reply]
Or simply peaking, cognitive bias is a cognitive bias itself.

**Comment by AttackOnMorty** [comment]
> Any good ideas to avoid it?

It's everywhere and you can't avoid it. I'd say how to deal with it.
- Set your own anchor, 
- Get more information to adjust it dynamically
- Try to reach an agreement.

**Comment by MattParkerDev** [comment]
Clients are always looking for anchors - we need to be tactful in avoiding providing inaccurate estimates/anchors prematurely.

**Comment by hveraus** [comment]
good stuff, it's also important to use it and be aware of its impact on decision-making to avoid manipulative practices and ensure fair outcomes.

**Comment by steven0x51** [comment]
anchoring can be a useful strategy or a bad thing, depends how we use it.

**Comment by jerryluossw** [comment]
> How do you feel about anchoring inside where you work?

 I think anchoring is very important when we talk to clients or leads.

> Does it happen much?

Yes.

> Any good ideas to avoid it?

I think we cannot avoid especially when we talk to people from different fields, what we should consider is how to deal with it better.

**Comment by Aibono1225** [comment]
Good concept. It's important to hear everyone's opinion before making any decisions.

**Comment by landonmaxwell** [comment]
The rule makes it sound like anchoring is inherently negative, which I disagree with.
Like a knife, it is a tool that can be incredibly useful but should be used with consideration.

> How do you feel about anchoring inside where you work?

Anchors prevented ships from crashing in safe harbours, but are dangerous in open waters.
Know when it is appropriate to anchor information and practice consciously both using & avoiding anchoring language.

> Does it happen much?

Yes. It's impossible to remove biases, as even the order that information & context is given will affect how it is received.
E.g. People usually say the information they think is most important first, so whatever they say first will unconsciously influence the proceeding information.

> Any good ideas to avoid it?

Check your adjectives - the connotations / implications of a word will anchor information.
E.g. 
- "can infiltrate various aspects of our lives" (negative)
- "can be found in various aspects of our lives" (neutral)
- "can fortify various aspects of our lives" (positive)

**Comment by jimmidier** [comment]
Having anchoring effect on what one perceives is a psychological defense mechanism that one tries to deny the mistakes that could potentially be made, which is unavoidable because it's human nature. On the other hand, as consultants we provide clients with information that could cause anchoring effect on them which is exactly why SSW has the rule to do "checked by" internally before delivering to the outside, to mitigate negative impacts of anchoring beforehand.

**Comment by tino-liu** [comment]
Anchoring could be useful when it comes to client. Because at certain times, knowledge/ideas should be imparted with a bit of bias leaning towards the person with the most technical skills.

But always go into a conversation with your own anchor, and also be flexible and learn from some else's take.

**Comment by AnkyChe** [comment]
Anchoring bias a double-edged sword when it comes to its impacts in the workplace. It's important to be aware of when anchoring is helpful versus when it becomes detrimental.

**Comment by babakamyljanovssw** [comment]
Interesting topic and good video. Always enjoy listening to Lex Fridman's podcasts. After reading other opinions I was able to understand the Anchoring concept. I think trying to be self aware and considering multiple sources of information can mitigate the anchoring effect.

**Comment by KristenHu** [comment]
> Any good ideas to avoid it?

I think using questions to gather more information and delaying judgment is a great way to avoid anchoring bias

**Comment by Freego1783** [comment]
I think anchoring can be both negative and positive depending on the situation. Just need to be careful and know how to work with it.

**Comment by tkapa** [comment]
I think that maybe there’s something to be said about potential benefits to anchoring. It’s difficult to come up with examples of it looking back. Though I’m sure that its happened to me.

**Comment by danielmackay** [comment]
I found this rule to be vague (but perhaps it's just me) and it didn't seem to line up with the same message from the video (i.e. we need honesty in meetings, get juniors to speak first, and look for data to back up decisions).

**Comment by PothieuG** [comment]
Knowing how anchoring bias works significantly helps in managing it :)

**Comment by PennyWalker** [comment]
I see this a lot in recruitment, if I don't ask people to send an /10 through at the same time, people will always align with whoever goes first,  and if it's someone senior it's likely to influence the figure. I like the idea of letting juniors speak first so they feel confident in speaking up, as often they don't want to go against the more senior person and it might be a great idea they wouldn't otherwise say.

Anchoring in Sales is a bigger problem when people hold onto a figure just because it was mentioned at some point, very dangerous if it's not a true estimate.

**Comment by Marxoz** [comment]
I don't think using estimates for a project is a good example to explain anchoring.

**Comment by vladislav-kir** [comment]
Anchoring, and dealing with biases, is very important. Early incorrect estimates can lead to disasters.
